src/components/PageContainer/PageContainer.module.css

```diff
@@ -29,4 +29,20 @@
 .exitButton{
   display: flex;
   justify-content: flex-end;
+}
+
+.closeButton{
+  border-radius: 50%;
+  background-color: #FFFFFF;
+  border: none;
+  padding: 0px;
+  height: 36px;
+  padding-top: 3px;
```

Comment:
Why the mix of padding and margin?
Why not just stick to one?

Reply:
They do different stuff and if I don't use both the x will not be centered. It's a result of trial and error, not any calculations. It's a little bit messy and if this causes problems for us in future we should probably make UX draw up a red x-icon that fits this size of button, with the exactly the right viewbox to fit this size of button (as this is something that also affects the centering of the x-icon). I just thought this was a static button that's only going to be used in this exact size and we probably won't adjust the sizes later on.